xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DeleteAction.java

```diff
@@ -226,12 +231,39 @@ private boolean deleteToRecycleBin(XWikiContext context) throws XWikiException
             doesAffectChildren(request, doc.getDocumentReference()) ? doc.getDocumentReference().getLastSpaceReference()
                 : doc.getTranslatedDocument(context).getDocumentReferenceWithLocale();
 
-        return deleteToRecycleBin(documentReference, context);
+        return deleteToRecycleBin(documentReference, context, skipRecycleBin);
+    }
+
+    /**
+     *
+     * @param entityReference the entity to delete
+     * @param context the current content
```

Comment:
typo

Reply:
BTW that comment is not wildly interesting ;)

Reply:
Documentation updated https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-70855079f42aabba98e87755fa9e71ecR254